app/src/backend/formulas.ts

```diff
@@ -61,6 +72,189 @@ export function handleDirectMeasureFormula(
   return gases;
 }
 
+export async function handleIncinerationWasteFormula(
+  activityValue: ActivityValue,
+  inventoryValue: InventoryValue,
+): Promise<Gas[]> {
+  const data = activityValue.activityData;
+
+  const CO2OxidationFactor: Record<string, number> = {
+    "technology-continuous-incineration": 1,
+    "technology-semi-continuous-incineration": 1,
+    "technology-batch-type-incineration": 1,
+    "technology-open-burning": 0.58,
+  };
+
+  const CH4EmissionFactor: Record<string, Record<string, number>> = {
+    "technology-continuous-incineration": {
+      "boiler-type-stoker": 0.2,
+      "boiler-type-fluidised-bed": 0,
+    },
+    "technology-semi-continuous-incineration": {
+      "boiler-type-stoker": 6,
+      "boiler-type-fluidised-bed": 188,
+    },
+    "technology-batch-type-incineration": {
+      "boiler-type-stoker": 60,
+      "boiler-type-fluidised-bed": 237,
+    },
+  };
+
+  const NO2EmissionFactor: Record<string, Record<string, number>> = {
+    "waste-composition-municipal-solid-waste": {
+      "technology-continuous-incineration": 50,
+      "technology-semi-continuous-incineration": 50,
+      "technology-batch-type-incineration": 50,
+      "technology-open-burning": 150,
+    },
+    "waste-composition-industrial-solid-waste": {
+      "technology-continuous-incineration": 100,
+      "technology-semi-continuous-incineration": 100,
+      "technology-batch-type-incineration": 100,
+      "technology-open-burning": 100,
+    },
+    "waste-composition-clinical-waste": {
+      "technology-continuous-incineration": 100,
+      "technology-semi-continuous-incineration": 100,
+      "technology-batch-type-incineration": 100,
+      "technology-open-burning": 100,
+    },
+    "waste-composition-hazardous-waste": {
+      "technology-continuous-incineration": 100,
+      "technology-semi-continuous-incineration": 100,
+      "technology-batch-type-incineration": 100,
+      "technology-open-burning": 100,
+    },
+    "waste-composition-sewage-sludge": {
+      "technology-continuous-incineration": 900,
+      "technology-semi-continuous-incineration": 900,
+      "technology-batch-type-incineration": 900,
+      "technology-open-burning": 900,
+    },
+  };
+
+  if (!data) {
+    throw new createHttpError.BadRequest(
+      "Activity has no data associated, so it can't use 'incineration-waste' formula",
+    );
+  }
+
+  const activityTitle = activityValue.metadata?.["activityTitle"];
```

Comment:
Is this the correct key for it? Shouldn't it be saved to `activity-title`?

Reply:
no that was for the activity data. We can change this to snake case in the future